styleguide/content/brand-identity/page-patterns.md

```diff
@@ -0,0 +1,131 @@
+Establishing standards on spacing is a key element in maintaining consistency between different sections of a large web app. Having a limited set of spacing options is meant to help promote a good sense of proportion among the components, and also helps engineers avoid hard-coding padding measurements and allows them to use CSS variables from our spacing stylesheet.
+
+### Spacing Options
+
+We use a “t-shirt size” scale for specifying spacing/margins. The following options have corresponding variables in the spacing CSS stylesheet.
+
+<style>
```

Comment:
Should we pull these into the CSS file for our static content?

Reply:
I think so. I couldn't get it to work in ten minutes, so I decided to get everything else done. I'll see if I can get that to work